node/src/components/consensus/era_supervisor.rs

```diff
@@ -338,6 +341,11 @@ where
             .expect("current era does not exist")
     }
 
+    /// Returns `true` if the specified era is active and bonded.
+    fn is_bonded(&self, era_id: EraId) -> bool {
+        era_id.0 + BONDED_ERAS >= self.current_era.0 && era_id <= self.current_era
```

Comment:
Shouldn't it be a disjunction?

Reply:
No, `era` is bonded if `current ≥ era ≥ current - BONDED_ERAS`, i.e. if both inequalities are satisfied.

Reply:
Right, sorry.

Everywhere `is_bonded` is called we do the negation. I think `is_unbonded` would have easier simpler to read condition as well  `era > current && era - BONDED_ERAS < current`

Reply:
Not sure… `is_bonded` is kind of the more natural function, isn't it?

Reply:
Yeah, you're right. It's weird that all usages are negating the result. Don't worry, it was just an opinion.